plugin/inc/views/tickets.class.php

```diff
@@ -406,6 +433,15 @@ protected function getSettings(): array
                                    WHERE (codl = 'F' OR stare like 'V%') AND tip like 'TF%'
                                    AND valinc-valpla > 0
                                    GROUP BY codbenef) t2 ON t2.codbenef = s.hmarfa_code_field
+        ";
+
+        $version      = str_replace('V', '', IserviceToolBox::getInputVariable('v', 1));
+        $otherVersion = $version == 1 ? 2 : 1;
+
+        $settingsV1 = [
+            'name' => self::getName() . " V$version",
```

Comment:
1. I would not use "V" in the variable value
2. I would use small "v" to display the version number (The button for v2 counters has the following label: "Contoare aparate v2").
```suggestion
        $version      = intval(IserviceToolBox::getInputVariable('v', 1));
        $otherVersion = $version === 1 ? 2 : 1;

        $settingsV1 = [
            'name' => self::getName() . " v$version",
```